scripts/mirror.js

```diff
@@ -0,0 +1,609 @@
+};
+
+/**
+ * @param {Identifier} data
+ * @param {string} feature
+ * @param {string} browser
+ * @param {string} source
+ * @param {string} modify
+ * @returns {Identifier}
+ */
+const setFeature = (data, feature, browser, source, modify) => {
+  let newData = Object.assign({}, data);
+
+  const rootPath = feature.shift();
+  if (feature.length > 0 && data[rootPath].constructor == Object) {
+    newData[rootPath] = setFeature(
+      data[rootPath],
+      feature,
+      browser,
+      source,
+      modify,
+    );
+  } else {
+    if (data[rootPath].constructor == Object || Array.isArray(data[rootPath])) {
+      newData = doSetFeature(data, newData, rootPath, browser, source, modify);
+    }
+  }
+
+  return newData;
+};
+
+/**
+ * @param {Identifier} data
+ * @param {string} browser
+ * @param {string} source
+ * @param {string} modify
+ * @returns {Identifier}
+ */
+const setFeatureRecursive = (data, browser, source, modify) => {
+  let newData = Object.assign({}, data);
+
+  for (let i in data) {
+    if (!!data[i] && typeof data[i] == 'object' && i !== '__compat') {
+      newData[i] = data[i];
+      if (data[i].__compat) {
+        doSetFeature(data, newData, i, browser, source, modify);
+      }
+      setFeatureRecursive(data[i], browser, source, modify);
+    }
+  }
+
+  return newData;
+};
+
+/**
+ * @param {string} browser
+ * @param {string} filepath
+ * @param {string} source
+ * @param {string} modify
+ * @returns {boolean}
+ */
+function mirrorDataByFile(browser, filepath, source, modify) {
+  let file = filepath;
+  if (file.indexOf(__dirname) !== 0) {
+    file = path.resolve(__dirname, '..', file);
+  }
+
+  if (!fs.existsSync(file)) {
+    return false;
+  }
+
+  if (fs.statSync(file).isFile()) {
+    if (path.extname(file) === '.json') {
+      let data = require(file);
+      let newData = setFeatureRecursive(data, browser, source, modify);
+
+      fs.writeFileSync(file, JSON.stringify(newData, null, 2) + '\n', 'utf-8');
+    }
+  } else if (fs.statSync(file).isDirectory()) {
+    const subFiles = fs.readdirSync(file).map(subfile => {
+      return path.join(file, subfile);
+    });
+
+    for (let subfile of subFiles) {
+      mirrorDataByFile(browser, subfile, source, modify);
+    }
+  }
+
+  return true;
+}
+
+/**
+ * @param {string} browser
+ * @param {string} featureIdent
+ * @param {string} source
+ * @param {string} modify
+ * @returns {boolean}
+ */
+const mirrorDataByFeature = (browser, featureIdent, source, modify) => {
+  let filepath = path.resolve(__dirname, '..');
+  let feature = featureIdent.split('.');
+  let found = false;
+
+  for (let depth = 0; depth < feature.length; depth++) {
+    filepath = path.resolve(filepath, feature[depth]);
+    const filename = filepath + '.json';
+    if (fs.existsSync(filename) && fs.statSync(filename).isFile()) {
+      filepath = filename;
+      found = true;
+      break;
+    }
+  }
+
+  if (!found) {
+    console.error(`Could not find ${feature}!`);
```

Comment:
Right now this outputs:
"Could not find html,elements,input,input-button!"
better would be:
"Could not find file for html.elements.input-button!"

```suggestion
    console.error(`Could not find file for ${featureIdent}!`);
```